### PR TITLE
fixed moons

### PR DESCRIPTION
**Pull request by cholakovvv**

### CODE REVIEW

**Review by petya0111 — APPROVED** [review]
Perfect moons! To the moon 🚀

File: client/src/basicExamples/sass/SassExample.module.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.text {
+  color: tomato;
+  font-size: 24px;
+}/*# sourceMappingURL=SassExample.module.css.map */
```
Comment (IvayloSlavchev) [inline client/src/basicExamples/sass/SassExample.module.css]: You can remove this comment.

**Review by IvayloSlavchev — APPROVED** [review]
Great job, @SCholakov05!

**Comment by ZahariCheyrekov** [comment]
LGTM!

**Pull request merged**